providers/openfeature-go-feature-flag-provider/README.md

```diff
@@ -0,0 +1,133 @@
+<p align="center">
+  <img width="400" src="https://raw.githubusercontent.com/thomaspoignant/go-feature-flag/main/gofeatureflag.svg" alt="go-feature-flag logo" />
+
+</p>
+
+# GO Feature Flag - OpenFeature Ruby provider
+<p align="center">
+   <a href="https://github.com/open-feature/ruby-sdk-contrib/tree/main/providers/openfeature-go-feature-flag-provider"><img src="https://img.shields.io/gem/v/openfeature-go-feature-flag-provider?color=blue&style=flat-square&logo=ruby" alt="gem"></a>
+   <a href="https://gofeatureflag.org/"><img src="https://img.shields.io/badge/%F0%9F%93%92-Website-blue" alt="Documentation"></a>
+   <a href="https://github.com/thomaspoignant/go-feature-flag/issues"><img src="https://img.shields.io/badge/%E2%9C%8F%EF%B8%8F-issues-red" alt="Issues"></a>
+   <a href="https://gofeatureflag.org/slack"><img src="https://img.shields.io/badge/join-us%20on%20slack-gray.svg?longCache=true&logo=slack&colorB=green" alt="Join us on slack"></a>
+</p>
+
+This repository contains the official Ruby OpenFeature provider for accessing your feature flags with [GO Feature Flag](https://gofeatureflag.org).
+
+In conjuction with the [OpenFeature SDK](https://openfeature.dev/docs/reference/concepts/provider) you will be able
+to evaluate your feature flags in your Ruby applications.
+
+For documentation related to flags management in GO Feature Flag,
+refer to the [GO Feature Flag documentation website](https://gofeatureflag.org/docs).
+
+### Functionalities:
+- Manage the integration of the OpenFeature Ruby SDK and GO Feature Flag relay-proxy.
+
+## Dependency Setup
+
+### Gem Package Manager
+
+Add this line to your application's Gemfile:
+```
+gem 'openfeature-go-feature-flag-provider'
+```
+And then execute:
+```
+bundle install
+```
+Or install it yourself as:
+```
+gem install openfeature-go-feature-flag-provider
+```
+
+## Getting started
+
+### Initialize the provider
+
+The `OpenFeature::GoFeatureFlag::Provider` needs some options to be created and then set in the OpenFeature SDK.
+
+| **Option** | **Description**                                                                                                                             |
+|------------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| `endpoint` | **(mandatory)** The URL to access to the relay-proxy.<br />*(example: `https://relay.proxy.gofeatureflag.org/`)*                            |
+| `headers`  | A `Hash` object containing the headers to send to the relay-proxy.<br/>*(example to send APIKey: `{"Authorization" => "Bearer my-api-key"}` |
+
+The only required option to create a `GoFeatureFlagProvider` is the URL _(`endpoint`)_ to your GO Feature Flag relay-proxy instance.
+
+```ruby
+import GOFeatureFlag
+import OpenFeature
+
+# ...
+
```

Comment:
I don't think this is valid ruby, you can probably remove this as I expect the constants will have been required by bundler